api/src/routes/documentation.ts

```diff
@@ -0,0 +1,27 @@
+import { FastifyPluginCallbackTypebox } from '@fastify/type-provider-typebox';
+import { FastifyInstance, FastifyReply, FastifySchema } from 'fastify';
+
+import databaseJsonSchema from '../../database-schemas/json-schema.json';
+
+import { UpdateReqType } from '../utils';
+
+// eslint-disable-next-line jsdoc/require-param
+/**
+ * Additional GET routes for documentation about this API.
+ */
+export const documentationRoutes: FastifyPluginCallbackTypebox = (
+  fastify,
+  _options,
+  done
+) => {
+  fastify.get('/documentation/db-schema', getDatabaseSchema);
+  done();
+};
+
+async function getDatabaseSchema(
+  this: FastifyInstance,
+  _req: UpdateReqType<FastifySchema>,
+  reply: FastifyReply
+) {
+  return reply.send(databaseJsonSchema);
+}
```

Comment:
```suggestion
  fastify.get('/documentation/db-schema', function (_req, reply) {
    void reply.send(databaseJsonSchema);
  });
  done();
};
```
Maybe? Seems cleaner.

Reply:
I specifically did this, because, whilst this is fine for a single endpoint, the minute we add more, it becomes annoying to see where the endpoints are.

Reply:
Okay, I would defer solving that problem until we have to, but it's only a style nitpick :shrug: